src/BlazorUI/Bit.BlazorUI/Components/Inputs/ColorPicker/BitColorPicker.scss

```
@@ -67,44 +141,72 @@
     position: relative;
     top: spacing(-0.0625);
     -webkit-appearance: none;
-    margin: 0 spacing(-0.875);
-    width: calc(100% + #{spacing(1.75)});
+    margin: 0 calc(var(--bit-clp-thumb-size) / -2);
+    width: calc(100% + var(--bit-clp-thumb-size));
+    height: var(--bit-clp-track-height);
+
+    &:focus-visible {
+        @include focus-ring;
+        border-radius: spacing(0.25);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)